net/net.go

```diff
@@ -50,16 +50,14 @@ func RemoteHost(r *http.Request) net.IP {
 //
 //     X-Forwarded-For: ip-address-1, ip-address-2, client-ip-address
 func RemoteHostFromLast(r *http.Request) net.IP {
-	a := r.Header["X-Forwarded-For"]
-	if a == nil {
-		return parse(r.RemoteAddr)
-	}
-	l := len(a) - 1
-	if l < 0 {
-		l = 0
-	}
-	if ffh := parse(a[l]); ffh != nil {
-		return ffh
+	ffs := r.Header.Get("X-Forwarded-For")
+	ffa := strings.Split(ffs, ",")
+	ff := ffa[len(ffa)-1]
+	if ff != "" {
```

Comment:
i think it would be better to trim the space before checking for empty. What do you think?

Reply:
We need to trim the words as far as I tested these have `, `